### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -6,13 +6,53 @@ packages:
 
 parts:
   - translation:
-      key: "txt.apps.infusionsoft_app.app.description"
-      title: "Application Description"
+      key: "txt.apps.infusionsoft_app.app.name"
+      title: "Name for App: Infusionsoft"
+      value: "Infusionsoft"
```
Comment (injuhwang) [inline translations/en.yml]: Please add `hidden: true`.

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍 for the strings

**Comment by pmgrove** [comment]
@svizzari: 💇

**Pull request merged**